lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -40,6 +42,16 @@ function divideByGCD(fraction) {
     return Node.Status.noChange(fraction);
   }
 
+  const intermediateNumerator = Node.Creator.operator(
+    '/', [Node.Creator.constant(numeratorValue), Node.Creator.constant(gcd)]);
+  const intermediateDenominator = Node.Creator.operator(
+    '/', [Node.Creator.constant(denominatorValue), Node.Creator.constant(gcd)]);
+  const intermediateFraction = Node.Creator.operator(
+    '/', [intermediateNumerator, intermediateDenominator]);
+  const status = Node.Status.nodeChanged(
+    ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  substeps.push(status);
```

Comment:
Usually we call `resetChangeGroups` after this but I'm not sure if that's needed here?